src/views/waterUse/WaterUse.vue

```diff
@@ -8,31 +8,45 @@
         </h1>
       </div>
     </div>
-    <div v-if="!showNewFeatureTabs">
```

Comment:
This section is for warnings related to the Beta tier. So we do want to keep it. If your run 'npm run serve-beta'
you will see how they work. The div should encapsulate all the visuals on the page as so that those won't show up by to the public, until we are ready to take that risk.